pyronear/datasets/openfire.py

```diff
@@ -17,6 +17,18 @@
 
 class OpenFire(VisionDataset):
     """Wildfire image Dataset.
+    Creates the directory structure below and downloads the dataset or use the existing
+    files
+    
+    - root/
+    | - extract.json
+    | - OpenFire/
+    | | - processed/
+    | | |  - test.pl
+    | | |  - training.pl
```

Comment:
Little typo here: `.pt` extensions, not `.pl`

Reply:
Thanks, fixed